src/layout/menu/ExternalLink.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import { screen } from '@testing-library/react';
+import { expect } from '@jest/globals';
+import { CustomRenderReactIntl } from 'test-utils/CustomRenderReactIntl';
+import { ExternalLink } from 'layout/menu/ExternalLink';
+import { messages } from 'lib/i18n/localization';
+
+describe('ExternalLink', () => {
+    it('should render the component', async () => {
+        CustomRenderReactIntl(<ExternalLink href="https://xitaso.com/" descriptor={messages.mnestix.imprint} />);
+        const component = screen.getByTestId('external-link');
+        expect(component).toBeInTheDocument();
+    });
+
+    it('should open link in new tab natively', async () => {
```

Comment:
I could not find a way to actually test, if a new tab is opened.
The `target="_blank"` attribute will use the native browser feature to open in a new tab and will not use `window.open`, so we cannot mock it